core/modules/parser/QSMySqlListener.cc

```diff
@@ -1224,6 +1224,8 @@ class BinaryComparasionPredicateAdapter :
             compPredicate->op = SqlSQL2Tokens::LESS_THAN_OP;
         } else if ("<>" == _comparison) {
             compPredicate->op = SqlSQL2Tokens::NOT_EQUALS_OP;
+        } else if ("!=" == _comparison) {
```

Comment:
Do you need to support '<=' and '>=' as well?

Reply:
it hasn't come up yet. (there's no test for it, so right now: no)